backend/migrations/sqls/20211123150850-addStudentIdAndStudentNameToAbiturpruefungen-down.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE abiturpruefungen_backup AS SELECT id, betreuendeLehrkraftID, art, bezugsfachID, genehmigt, ablehnungsgrund, partnerStudentId, referenzfachID, thema FROM abiturpruefungen;
```

Comment:
Damit gehen die Datentypen verloren. Ob du das anpasst überlass ich dir

Reply:
Jetzt wie in addPartnerStudentIdToAbiturpruefungen-down.sql